charts/spire/charts/spire-server/values.yaml

```
@@ -170,7 +170,6 @@ ca_subject:
   country: NL
   organization: Example
   common_name: example.org
```

Blank line removed by mistake. suggestion to add it back.
```suggestion
  common_name: example.org
  
```